scripts/azure-pipelines/linux-check.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+##############################################################################
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+set -x
+
+function or_die () {
+    "$@"
+    local status=$?
+    if [[ $status != 0 ]] ; then
+        echo ERROR $status command: $@
+        exit $status
+    fi
+}
+
+or_die cd axom
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+echo "~~~~~~ RUNNING CMAKE ~~~~~~~~"
+or_die ./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake
+or_die cd build-$HOST_CONFIG-debug
+echo "~~~~~~ RUNNING make check ~~~~~~~~"
+or_die make check
```

Comment:
Is `make check` passing for you? 

Last time I tried `cppcheck` was segfaulting in several of our files.

(Checking to see if we have an issue for this).

Reply:
no idea.. i cant run w/o the docker image pushed.  I figured I would wing it =)

Reply:
https://github.com/LLNL/axom/issues/368

_(I definitely reported that the first time I saw it, and not 2 minutes ago)_

Reply:
totally.. i was there. I can back you up.